src/component/Table.js

```diff
@@ -0,0 +1,10 @@
+const Table = ({cards,name}) => {
+  cards = cards.map((card) => <div key={card.value+card.suit}>{card.value}{card.suit}</div>);
```

Comment:
É sempre bom evitar concatenação de string dessa forma, prefira usar [strint interpolation/template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)

```suggestion
  cards = cards.map((card) => <div key={`${card.value}${card.suit}`}>{card.value}{card.suit}</div>);
```